pkg/webhook/policy.go

```diff
@@ -480,6 +509,81 @@ func (h *validationHandler) reviewRequest(ctx context.Context, req *admission.Re
 	}
 
 	trace, dump := h.tracingLevel(ctx, req)
+	review, err := h.createReview(ctx, req)
+
+	// Check if the requested resource is a generator resource that is configured
+	// to be expanded by TemplateExpansions
+	gvk := schema.GroupVersionKind{
+		Group:   req.Kind.Group,
+		Version: req.Kind.Version,
+		Kind:    req.Kind.Kind,
+	}
+	expansionTemps := h.expansionSystem.TemplatesForGVK(gvk)
+	// If there are no matching ExpansionTemplates, review the resource normally
+	if len(expansionTemps) == 0 {
```

Comment:
there is no need to special-case this code, the original object gets reviewed regardless.